src/main/scala/no/ndla/imageapi/ImageApiProperties.scala

```diff
@@ -34,6 +34,18 @@ object ImageApiProperties extends LazyLogging {
 
   val DefaultLanguage = "nb"
 
+  val oldCreatorTypes = List("opphavsmann", "fotograf", "kunstner", "redaksjonelt", "forfatter", "manusforfatter", "innleser", "oversetter", "regissør", "illustratør", "medforfatter", "komponist")
```

Comment:
Vet ikke om det er best å bruke *Properties filene eller mapping saken for de her jeg(?)

Reply:
Tenker det er greit å ha dem i *Properties filene slik du har gjort. De er vel bare brukt ifbm. import så langt jeg ser, og tror det er lurt å holde biblioteker fritt fra detaljer fra gammelt system

Reply:
De blir også brukt i `ValidationService.validateAuthorType` (ikke `old*Types`, men de andre)